client/log_client.go

```diff
@@ -399,10 +399,15 @@ func (c *LogClient) AddSequencedLeaves(ctx context.Context, dataByIndex map[int6
 		leaf.LeafIndex = index
 		leaves = append(leaves, leaf)
 	}
-	_, err := c.client.AddSequencedLeaves(ctx, &trillian.AddSequencedLeavesRequest{
+	resp, err := c.client.AddSequencedLeaves(ctx, &trillian.AddSequencedLeavesRequest{
 		LogId:  c.LogID,
 		Leaves: leaves,
 	})
+	for _, leaf := range resp.GetResults() {
+		if stat := leaf.GetStatus().GetCode(); stat != int32(codes.OK) && stat != int32(codes.AlreadyExists) {
```

Comment:
It might be easier to use the [gRPC status](https://github.com/grpc/grpc-go/blob/master/status/status.go#L119) helper functions here:

```suggestion
		if s := status.FromProto(leaf.GetStatus()); s.Code() != codes.OK && s.Code() != codes.AlreadyExists {
```